spring-kafka/src/main/java/org/springframework/kafka/requestreply/ReplyingKafkaTemplate.java

```diff
@@ -100,6 +104,38 @@ public void setReplyTimeout(long replyTimeout) {
 		this.replyTimeout = replyTimeout;
 	}
 
+	@Override
+	public boolean isRunning() {
+		return this.running;
+	}
+
+	@Override
+	public int getPhase() {
+		return this.phase;
+	}
+
+	public void setPhase(int phase) {
+		this.phase = phase;
+	}
+
+	@Override
+	public boolean isAutoStartup() {
+		return this.autoStartup;
+	}
+
+	public void setAutoStartup(boolean autoStartup) {
+		this.autoStartup = autoStartup;
+	}
+
+	/**
+	 * Return the topics configured for the replying listener container.
+	 * @param patternConsumer called if the container is subscribed to a pattern.
+	 * @return the topics or an empty list if the container is subscribed to a pattern.
+	 */
+	public Collection<TopicPartition> obtainReplyTopics(Consumer<Pattern> patternConsumer) {
```

Comment:
I think you are in a hurry.
The `patternConsumer` is redundant and misleading now.

Plus we have talked about returning *topic names* for the reply correlation logic (I guess).
But now you return full `TopicPartition`.

Well, that might be useful on the end-user side, indeed, but let's rename the method then!